org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/services/extensions/HTMLCompletionExtensionsTest.java

```diff
@@ -118,6 +119,19 @@ public void testHTMLAttributeNameCompletion() throws BadLocationException {
 	public void testHTMLAttributeValueCompletion() throws BadLocationException {
 		testCompletionFor("<input type=|", c("text", "\"text\""/* "<input type=\"text\"" */), //
 				c("checkbox", "\"checkbox\"" /* "<input type=\"checkbox\"" */));
+		
+		// TODO: Move below to testHTMLOnXMLContentCompletion()
+		CompletionItem completion = new CompletionItem("Test replace range");
+		Position start = new Position(0, 7);
+		Position end = new Position(0, 26);
+		TextEdit edit = new TextEdit(new Range(start, end), "replacement text");
+		completion.setTextEdit(edit);
+		testCompletionFor("<input>some extisti|ng text</input>", completion );
+	}
+	
+	@Test
+	private void testHTMLOnXMLContentCompletion() throws BadLocationException {
+		// TODO: This isin't getting called by Junit

```

Comment:
I don't know why I couldn't get my test to be called :P Is there anything I'm missing to get it called? For now I put the test code in `testHTMLAttributeValueCompletion` so that it'd be called on CI

Reply:
needs to be public